Epic 3 - ReportApp/ReportApp/Services/Activity/ActivityConfigurationLoaderService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using ReportApp.Models.ActivityReport;
+
+namespace ReportApp.Services.Activity;
+
+public class ActivityConfigurationLoaderService : IConfigurationLoader<ActivityReportConfiguration>
+{
+    public ActivityReportConfiguration LoadConfiguration(string path)
```

Comment:
Make sense to make this method virtual/override?

Reply:
I believe this will simplify your code and remove the generic, which is unnecessary here. Also it will improve dependency registration